app/vminsert/csvimport/request_handler.go

```diff
@@ -40,14 +40,14 @@ func insertRows(rows []parser.Row, extraLabels []prompbmarshal.Label) error {
 		r := &rows[i]
 		ctx.Labels = ctx.Labels[:0]
 		ctx.AddLabel("", r.Metric)
-		for j := range r.Tags {
-			tag := &r.Tags[j]
-			ctx.AddLabel(tag.Key, tag.Value)
-		}
 		for j := range extraLabels {
```

Comment:
`extraLabels` must be added after adding user-provided labels, so they could override user-provided labels according to the updated `MetricName.sortTags` functionality at c8ea697db8be2e427b5c1d0af5a67495d55cafda .

Reply:
updated

Reply:
yep, changed. Also added test cases.